lib/plugins/aws/package/compile/events/alb/lib/target.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+  getTargetId(functionName) {
+    const { provisionedConcurrency } = this.serverless.service.functions[functionName];
+    const lambdaLogicalId = this.provider.naming.getLambdaLogicalId(functionName);
+    const lambdaArn = { 'Fn::GetAtt': [lambdaLogicalId, 'Arn'], };
+    return provisionedConcurrency ?
+      { 'Fn::Join': [':', [lambdaArn, this.provider.naming.getLambdaProvisionedConcurrencyAliasName()]] }
+      : lambdaArn;
+  }
```

Comment:
Maybe it's possible to use AWS::Lambda::Alias returned value instead (would make the code cleaner), i will try this out.